src/models/AzureClientInfo.ts

```diff
@@ -0,0 +1,13 @@
+/*---------------------------------------------------------------------------------------------
```

Comment:
chould you add this to the subfolder and create a toplevel folder azure.

Reply:
add the function related contracts under models functions folder i.e. azure/models/functions